abb_irb120_support/urdf/irb120_3_58_macro.xacro

```diff
@@ -203,19 +202,30 @@
       <axis xyz="1 0 0"/>
       <dynamics damping="0.0" friction="0.0"/>
     </joint>
-    <joint name="${prefix}joint6-tool0" type="fixed">
-      <parent link="${prefix}link_6"/>
-      <child link="${prefix}tool0"/>
-      <origin rpy="0 0 0" xyz="0 0 0"/>
-    </joint>
     <!-- end of joint list -->
 
-    <!-- ROS base_link to ABB World Coordinates transform -->
+    <!-- ROS-Industrial 'base' frame: base_link to ABB World Coordinates transform -->
     <link name="${prefix}base" />
     <joint name="${prefix}base_link-base" type="fixed">
       <origin xyz="0 0 0" rpy="0 0 0"/>
       <parent link="${prefix}base_link"/>
       <child link="${prefix}base"/>
     </joint>
+
+    <!-- ROS-Industrial 'flange' frame: attachment point for EEF models -->
+    <link name="${prefix}flange"/>
+    <joint type="fixed" name="${prefix}joint_6-flange">
+      <origin xyz="0 0 0" rpy="0 0 0"/>
+      <parent link="${prefix}link_6"/>
+      <child link="${prefix}flange"/>
+    </joint>
+
+    <!-- ROS-Industrial 'tool0' frame: all-zeros tool frame -->
+    <link name="${prefix}tool0" />
+    <joint name="${prefix}link_6-tool0" type="fixed">
+      <origin xyz="0 0 0" rpy="0 ${radians(0)} 0" />
```

Comment:
I haven't checked yet, but is this `0` correct here?

Reply:
Have the same question actually. I just copied the original value in for now but will check in RobotStudio to make sure this is correct. Forgot to mention that in the PR. The same applies to `irb120t_3_58_macro.xacro`

Reply:
Have you had a chance to check this? I'm pretty sure this is incorrect.

However, we'll fix it in a later PR.

Reply:
Checked it just now and it was indeed an error. Added a commit to fix the error (https://github.com/ros-industrial/abb_experimental/pull/119/commits/0ae97e142ea6828c1f928f8525b6a9abc5635782)